.travis.yml

```diff
@@ -0,0 +1,19 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install: true
+install: true
+before_script: true
+script: true
+jobs:
+  include:
+    - stage: compliance
+      script: ci/rpm7/build.sh -c -p $(pwd)
+    - stage: unit
+      before_script: docker run -t --rm -v $(pwd):/opt/fiware-orion fiware/orion-ci:rpm7 build -M unit
```

Comment:
The `-b` parameter of `build` is not being used... thus, which branch is used to do the cheking? 

Just to be sure we are not "hardwiring" master branch here... the branch to check should be the one of the "candidate" PR in which travis will run.

Reply:
If -b parameter is not specified, source code will be taken from the specific folder. In "compliance" stage -p (target folder) defined because script run directly, not in the Docker container.

Reply:
NTC